assets/js/pages/UpgradablePackagesPage/UpgradablePackages.jsx

```diff
@@ -15,14 +17,67 @@ export default function UpgradablePackages({
   onLoad = noop,
 }) {
   const [search, setSearch] = useState('');
+  const enrichedPackages = upgradablePackages.map((packageDetails) => {
+    const { name, from_version, from_release, to_version, to_release, arch } =
+      packageDetails;
 
-  const displayedPackages = upgradablePackages.filter(
+    return {
+      ...packageDetails,
+      installed_package: `${name}-${from_version}-${from_release}.${arch}`,
+      latest_package: `${name}-${to_version}-${to_release}.${arch}`,
+    };
+  });
+  const displayedPackages = enrichedPackages.filter(
     ({ name, patches }) =>
       containsSubstring(name, search) ||
       patches
         .map(({ advisory }) => containsSubstring(advisory, search))
         .includes(true)
   );
+  const [csvURL, setCsvURL] = useState(null);
```

Comment:
nitpick: I would move this `useState` above, to be together with the other `useState`